contextual_orchestrator/server.py

```
@@ -5497,7 +5529,7 @@ def do_GET(self) -> None:  # noqa: N802
             except (TypeError, ValueError) as exc:
                 self._send_error(400, "invalid_request", str(exc))
             except Exception:
-                self._send_error(500, "internal_error", "internal server error")
+                import traceback; traceback.print_exc(); self._send_error(500, "internal_error", "internal server error")
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)